### ISSUE TITLE
Security Vulnerability in Veracode Scan

### ISSUE THREAD

**Issue by rkaartikeyan** [react-native-share/react-native-share#1212]
### Steps to reproduce
1. Build IPA
2. Scan with Veracode.

### Expected behaviour
The below vulnerabilities should not appear in the veracode scan.

### Actual behaviour
Part of publishing the apps, we were scan the IPA file with Veracode and veracode reported following vulnarabilities. Thanks for the great library, and if we can have fix for the below issues it will be great in security aspect too.

####1. WhatsAppShare.m
![image](https://user-images.githubusercontent.com/3778591/171870970-dc4652db-8f9e-4130-9fa1-ab745a6da44f.png)

```
Description: The NSURL.URLWithString:() method transmits sensitive or security-critical data in cleartext in a communication channel that can be sniffed by unauthorized actors.

Remediation: Transmit all sensitive information in encrypted form using SSL.
```

####2. InstagramStories.m
![image](https://user-images.githubusercontent.com/3778591/171871847-62f4be67-4aec-4723-9a5f-dc2710a96501.png)

```
Description: The NSURL.URLWithString:() method transmits sensitive or security-critical data in cleartext in a communication channel that can be sniffed by unauthorized actors.

Remediation: Transmit all sensitive information in encrypted form using SSL.

```

####3. FacebookStories.m
![image](https://user-images.githubusercontent.com/3778591/171872280-3bef641c-37aa-449d-8a8e-91cc40e5fb02.png)

```
Description: The NSURL.URLWithString:() method transmits sensitive or security-critical data in cleartext in a communication channel that can be sniffed by unauthorized actors.

Remediation: Transmit all sensitive information in encrypted form using SSL.
```

### Environment
- **React Native version**: 0.68
- **React Native platform + platform version**: iOS 11.0,
- **Typescript version** (if using typescript):  4.4.4

### react-native-share
**Version**: 7.3.7

**Comment by mikehardy** [comment]
Fantastic! Looks like an easy PR - can you try altering those to https and testing it to see if your app still shares correctly? You can just reach right in to node_modules and alter the files directly then rebuild

Super easy change, and the testing you'll perform is testing you need to perform anyway before we can release the change, that we can't perform for you,

https://hackernoon.com/i-thought-i-understood-open-source-i-was-wrong-cf54999c097b

You can use https://github.com/ds300/patch-package/ to make the changes persistent until the PR is merged in case that helps

**Comment by AliHaidar110** [comment]
[https://github.com/react-native-share/react-native-share/pull/1215](url)
pr for this issue

**Comment by MateusAndrade** [comment]
This was released with [7.6,1](https://github.com/react-native-share/react-native-share/releases/tag/v7.6.1). Can you test this @rkaartikeyan ?
